packages/core/src/Cardano/types/TxSubmissionErrors.ts

```diff
@@ -1,5 +1,6 @@
 import { CustomError } from 'ts-custom-error';
 import { TxSubmission } from '@cardano-ogmios/client';
+import _ from '@cardano-ogmios/schema';
```

Comment:
What was the issue here? :thinking: Seems like there's some difference in `tsc` (using different version in nix? not using the same `tsconfig.json`?)
```suggestion
import '@cardano-ogmios/schema';
```

Reply:
Some type is not imported from the schema, but used.

Reply:
`tsc` is exactly the same version as specified ...

Reply:
Would be great to understand what's the difference between the 2 environments that makes those extra imports needed :thinking:

Reply:
This seems like a workaround for bringing the Ogmios TypeDefs into scope, as this import is not required to satisfy the compiler given it can access it via `TxSubmission`.